src/lib/type.js

```diff
@@ -0,0 +1,24 @@
+import prisma from "~/lib/prisma";
+
+export const getAllTypes = () => {
+  return prisma.type.findMany({
+    select: {
+      id: true,
+      name: true,
+    },
+  });
+};
+
+export const getAllBreeds = () => {
```

Comment:
Can you rename this function `getBreeds()` for consistency with profiles and move it to its own file?